code/modules/mob/living/carbon/xenomorph/Castes/Crusher.dm

```diff
@@ -249,13 +249,13 @@
 	if(istype(target, /obj/structure/razorwire))
 		var/obj/structure/razorwire/B = target
 		if(charge_speed >= charge_speed_max) //plows right through
-			flags_pass = 0
+			flags_pass = TRUE
```

Comment:
```suggestion
			flags_pass |= PASSTABLE
```
passtable is defined as 1 which is why this works but use the proper flag define for the bitfield